### PR TITLE
Add EIP: Intent Abstract NFT

### PR DESCRIPTION
**Pull request by TsengMJ**
Simple Summary

This proposal defines Non-fungible Tokenized Interaction(NFTI), which enables NFT to own its defined logic and possesses composability.

### CODE REVIEW

File: EIPS/eip-7656.md [hunk 1]
```diff
@@ -0,0 +1,535 @@
+---
+eip: 7656
```
Comment (abcoathup) [inline EIPS/eip-7656.md:2]:
```suggestion
eip: 7513
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Comment (TsengMJ) [reply EIPS/eip-7656.md:2]: Thank you so much for your help, and I'm sorry for not noticing this earlier. I'd like to ask if there's any documentation regarding EIP numbering rules? Is it documented in EIP-1?

**Comment by xinbenlv** [comment]
Editorial: 
More clarity is needed:
1. Can we at least define "SmartNFT" or use a better term to refer to the contract being controlled with a ERC-1155 tokenId. That would make it more readable
2. Could the authors update the reference implementation because there are some confusion around the definition of "SmartNFT", the reference implementation could help explain what it means, who controls them and who could "use" them

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Pull request closed without merging**